src/drivers/MemoryDriver.ts

```diff
@@ -0,0 +1,57 @@
+import { IDriver } from "./IDriver";
+
+type Table = Map<string, any>;
```

Comment:
There is no need for this type since it is used only once on Line 6.
You can remove it and change Line 6 accordingly